src/schema/v2/fields/__tests__/money.test.ts

```diff
@@ -78,6 +78,23 @@ describe("amount", () => {
       `"$12.34"`
     )
   })
+
+  it("returns disambiguate symbol over symbol when optional param passed", () => {
+    expect(
+      getResult({
+        obj: { currencyCode: "USD", symbol: "$" },
+        args: { disambiguate: true },
+      })
+    ).toMatchInlineSnapshot(`"US$12.34"`)
+  })
+
+  it("returns symbol over disambiguate symbol by default", () => {
+    expect(
+      getResult({
+        obj: { currencyCode: "USD", symbol: "$" },
+      })
+    ).toMatchInlineSnapshot(`"$12.34"`)
+  })
```

Comment:
So based on https://github.com/artsy/metaphysics/pull/3692/files#r794203952, I think we can have more realistic specs to cover our use cases, which is without `symbol` on the object.

```typescript
  it("returns disambiguate symbol when optional param passed", () => {
    expect(
      getResult({
        obj: { currencyCode: "USD" },
        args: { disambiguate: true },
      })
    ).toMatchInlineSnapshot(`"US$12.34"`)
  })

  it("returns currency symbol by default", () => {
    expect(
      getResult({
        obj: { currencyCode: "USD" },
      })
    ).toMatchInlineSnapshot(`"$12.34"`)
  })
```